docs/topics/Opcodes_and_Status_Codes.md

```diff
@@ -149,6 +149,7 @@ Along with the HTTP error code, our API can also return more detailed error code
 | 50036 | An invite was accepted to a guild the application's bot is not in |
 | 50041 | Invalid API version |
 | 90001 | Reaction blocked |
+| XXXXX | Emojis cannot be larger than 256kb |
```

Comment:
this is a form error, not a normal api error, which unfortunately will not return an error code .. until we deploy new form error responses you won't be able to document it